src/providers/hana/qgshanadatatypes.h

```diff
@@ -0,0 +1,91 @@
+/***************************************************************************
+   qgshanadatatypes.h
+   --------------------------------------
+   Date      : 10-04-2024
+   Copyright : (C) SAP SE
+   Author    : Maxim Rylov
+ ***************************************************************************/
+
+/***************************************************************************
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ ***************************************************************************/
+#ifndef QGSHANADATATYPES_H
+#define QGSHANADATATYPES_H
+
+#include "odbc/Types.h"
+
+using namespace NS_ODBC;
+
+class QgsHanaDataTypes
+{
+    QgsHanaDataTypes() = delete;
+
+  public:
+    /// Unknown data type.
+    static constexpr int Unknown = SQLDataTypes::Unknown;
```

Comment:
Wouldn't this be better as a enum class instead?

Reply:
Hi @nyalldawson,
Yes, it is, but requires a bit more work.
I've considered your point in the last commit.